splitio/recorder/recorder.py

```diff
@@ -55,8 +55,9 @@ def __init__(self, impressions_manager, event_storage, impression_storage):
         self._impressions_manager = impressions_manager
         self._event_sotrage = event_storage
         self._impression_storage = impression_storage
+        self._telemetry_evaluation_producer = telemetry_evaluation_producer
 
-    def record_treatment_stats(self, impressions, latency, operation):
+    def record_treatment_stats(self, impressions, latency, operation, method_name):
```

Comment:
you can pass the required enum instead of the string

Reply:
updated the metric_name to use the telemetry enum